base/applications/rapps/installed.cpp

```diff
@@ -92,85 +59,7 @@ BOOL UninstallApplication(INT Index, BOOL bModify)
     return StartProcess(szPath, TRUE);
 }
 
-BOOL ShowInstalledAppInfo(INT Index)
-{
-    ATL::CStringW szText;
-    ATL::CStringW szInfo;
-    PINSTALLED_INFO Info = (PINSTALLED_INFO) ListViewGetlParam(Index);
-
-    if (!Info || !Info->hSubKey) return FALSE;
-
-    GetApplicationString(Info->hSubKey, L"DisplayName", szText);
```

Comment:
But it's not even needed anymore since `GetApplicationString()` is not used anywhere else :)